Doppler.HtmlEditorApi/Controllers/CampaignsController.cs

```diff
@@ -64,6 +64,17 @@ public Task<ActionResult> GetCampaignThumbnail(string accountName, int campaignI
         [HttpPut("/accounts/{accountName}/campaigns/{campaignId}/content")]
         public async Task<IActionResult> SaveCampaign(string accountName, int campaignId, CampaignContent campaignContent)
         {
+            // TODO: get this information from the configuration
+            var fieldAliases = new[]
+            {
+                new FieldAliasesDef("BIRTHDAY", new[] { "CUMPLEANOS", "CUMPLEAÑOS", "DATE OF BIRTH", "DOB", "FECHA DE NACIMIENTO", "NACIMIENTO" }),
+                new FieldAliasesDef("COUNTRY", new[] { "PAIS", "PAÍS" }),
+                new FieldAliasesDef("EMAIL", new[] { "CORREO", "CORREO ELECTRONICO", "CORREO ELECTRÓNICO", "CORREO_ELECTRONICO", "CORREO_ELECTRÓNICO", "E-MAIL", "MAIL" }),
+                new FieldAliasesDef("FIRST_NAME", new[] { "FIRST NAME", "FIRST-NAME", "FIRSTNAME", "NAME", "NOMBRE" }),
+                new FieldAliasesDef("GENDER", new[] { "GENERO", "GÉNERO", "SEXO" }),
+                new FieldAliasesDef("LAST_NAME", new[] { "LAST NAME", "LAST-NAME", "LASTNAME", "SURNAME", "APELLIDO" }),
```

Comment:
@andresmoschini , make sense considere the plural aliases for `NOMBRE` => `NOMBRES` and `APELLIDO` => `APELLIDOS`?

Reply:
Thanks @RodrigoPereyraDiaz 

It could be a good idea, but, by the moment I want to make honor to current Doppler's behavior (Doppler uses [this file](https://github.com/MakingSense/Doppler/blob/develop/Doppler.Application.ListsModule/Properties/FieldsRes.resx) to deal with the aliases).

If we change the behavior, we could introduce the risk that a user learns that _apellidos_ works in Unlayer Editor, and then he uses _apellidos_ in a zip to be imported. And it will not work there.